pkg/meta/model/job_args_test.go

```diff
@@ -613,3 +613,69 @@ func TestCheckConstraintArgs(t *testing.T) {
 		require.True(t, args.Enforced)
 	}
 }
+
+func TestLockTableArgs(t *testing.T) {
+	inArgs := &LockTablesArgs{
+		LockTables:    []TableLockTpInfo{{1, 1, model.TableLockNone}},
+		UnlockTables:  []TableLockTpInfo{{2, 2, model.TableLockNone}},
+		IndexOfLock:   0,
+		IndexOfUnlock: 0,
```

Comment:
```suggestion
		IndexOfLock:   34,
		IndexOfUnlock: 123,
```
zero is hard to difference with default